### ISSUE TITLE
observable:CyberItemRelationshipEnum does not appear to be tied to core:ControlledVocabulary

### ISSUE THREAD

**Issue by ajnelson-nist** [ucoProject/UCO#146]
It appears there should be a subclassing relationship.  Without it, I don't quite see how this snippet of JSON (drawn from an update I'm making to CASE's `[Oresteia.json](https://github.com/casework/CASE/blob/master/examples/Oresteia.json)`) would be valid with its `kindOfRelationship` statement.

```
{
   "@id": "trace-relationship3-uuid",
   "@type": "core:Relationship",
   "core:source": "sqlite_database",
   "core:target": "cassandra-image-partition6-uuid",
   "core:kindOfRelationship": {
     "@type": "observable:CyberItemRelationshipEnum",
     "@value": "Contained_Within"
   },
   "core:isDirectional": true
}
```

**Comment by sbarnum** [comment]
This should end up not being an issue once we resolve issue #75

Once the controlled vocabularies such as CyberItermRelationshipEnum are refactored to be defined as datatype based on string you should be able to convey the above JSON-LD and it will be valid.
What it lets us do is leave kindOfRelationship types as a general string and then local uses of that field can use JSON-LD typed values (https://json-ld.org/spec/latest/json-ld/#typed-values) to specify and coerce the local use of the property to be a more restricted type of string (in this case part of the specified controlled vocabulary). Another Relationship object in the same set of content could assert a totally different controlled vocabulary for its kindOfRelationship value.

**Comment by sbarnum** [comment]
Addressed in commit c92512e2660fbd880a9ddf09cbd7819f2f0334ff

Example JSON will need updated.

**Comment by sbarnum** [comment]
Fixed in https://github.com/ucoProject/UCO/releases/tag/0.4.0
